### BIG differences, this is diff of wrong files.
### CLUS "System Class COMPLEX"
### and CLHS topic selection, http://clhs.lisp.se/Body/a_comple.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:complex.txt	2017-04-27 21:49:59.000000000 +0100
+++ saved-pages-clus/cl:types:complex.txt	2017-04-27 21:49:59.000000000 +0100
@@ -1,4 +1,69 @@
-COMPLEX
-Please select which reference to COMPLEX you intended
+System Class COMPLEX

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Class Precedence List
+complex, number, t
+Description
+The type complex includes all mathematical complex numbers other than those included in the type rational. Complexes are
+expressed in Cartesian form with a real part and an imaginary part, each of which is a real. The real part and imaginary part are either both rational or both of the same float type. The imaginary part can be a float zero, but can never be a rational zero, for such a number is always represented by Common Lisp as a rational rather than a complex.
+Compound Type Specifier Kind
+Specializing.
+Compound Type Specifier Syntax
+ complex [typespec | *]
+Compound Type Specifier Arguments
+ typespec - a type specifier that denotes a subtype of real.
+Compound Type Specifier Description
+Every element of this type is a complex whose real part and imaginary part are each of type (upgraded-complex-part-type typespec).
+This type encompasses those complexes that can result by giving numbers of type typespec to complex.
+(complex type-specifier)]] refers to all complexes that can result from giving numbers of type type-specifier to the function complex, plus all other complexes of the same specialized representation.
+See Also
+ {\secref\RuleOfCanonRepForComplexRationals}
+ {\secref\NumsFromTokens}, {\secref\PrintingComplexes}
+Notes
+The input syntax for a complex with real part r and imaginary part i is #C(r i). For further details, see section {\secref\StandardMacroChars}.
+For every float, n, there is a complex which represents the same mathematical number and which can be obtained by (COERCE ''n'' 'COMPLEX).
+\
\ No newline at end of file
